test/configs/config.spec.ts

```diff
@@ -159,7 +159,7 @@ describe('AutoblocksConfig', () => {
       });
 
       await config.activateRemoteConfig({
-        config: {
+        remoteConfig: {
           id: 'my-config-id',
```

Comment:
just occurred to me, shouldn't the config ID be specified when initializing the `AutoblocksConfig` class? it wouldn't make sense to activate a remote config w/ different IDs from the same instance would it?

Reply:
so like this:

```ts
const config = new AutoblocksConfig<{ foo: string }>({
  id: 'my-config-id',
  values: { foo: 'bar' },
});

await config.activateRemoteConfig({
  version: {
    latest: true,
  },
  apiKey: '...',
});

// or 

await config.activateRemoteConfig({
  version: {
    dangerouslyUseUndeployed: {
      latest: true,
    },
  },
  apiKey: '...',
});
```

Reply:
oh i see, ya that is true